packages/store/src/data-provider/backend-system.ts

```diff
@@ -0,0 +1,57 @@
+import { ServiceOptions } from '../types';
+import { DataProvider, DataProviderConstructor } from '.';
+import { DataAccess } from '../data-access';
+import { HybridStore } from '../data-access/hybrid';
+import { BackendSystem, BackendSystemKey } from '../entities/backend-system';
+import { Logger } from '@sap-ux/logger';
+import { Entities } from './constants';
+
+export const SystemDataProvider: DataProviderConstructor<BackendSystem, BackendSystemKey> = class
```

Comment:
Can we add jsdoc here and to each function to explain what this does?

Reply:
I'm having trouble coming up with what can go in here without repeating what's in the code. It makes more sense to have some js-doc comments where the generic interface is defined.

Reply:
Added a comment in the interface in https://github.com/SAP/open-ux-tools/pull/237/commits/e37e1de9ea46e33d3a9e87c4febfd4187ff5186a